### PR TITLE
Support Symfony 7

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Opened this one in favor of #1326, #1328 and #1331.

The goal is to unite the work of all other PRs and debug this until we have the awaited compatibility 🙂 

The only remaining failure is strange, I don't understand it:

```
1) Enqueue\SnsQs\Tests\Spec\SnsQsSendToTopicAndReceiveNoWaitFromQueueTest::test
Failed asserting that null is an instance of interface "Interop\Queue\Message".

/mqdev/vendor/queue-interop/queue-spec/src/SendToTopicAndReceiveNoWaitFromQueueSpec.php:50
/mqdev/pkg/test/RetryTrait.php:26
```

I'm having a look at it but please share any info if you have an idea 😄 But I could not find any info about SNSQS, has it been dropped? 🤔 This would explain the failure.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -78,17 +78,19 @@ jobs:
...
           - php: '7.4'
-            symfony_version: '6.3.*'
+            symfony_version: '7.0.*'
           - php: '8.0'
-            symfony_version: '6.2.*'
+            symfony_version: '6.4.*'
           - php: '8.0'
-            symfony_version: '6.3.*'
+            symfony_version: '7.0.*'
+          - php: '8.1'
```
Comment (IlyaPokamestov) [inline .github/workflows/ci.yml:92]: Symfony 7 requires PHP 8.2 or upper: https://symfony.com/releases/7.0

```suggestion
          - php: '8.2'
```

Comment (alexandre-daubois) [reply .github/workflows/ci.yml:92]: This is the exclusion matrix, which points to PHP version that should **not** be run with this Symfony version. I think this is the correct setting here if I'm not missing something

**Comment by biobodya** [comment]
Hi, any news about PR?

**Comment by alexandre-daubois** [comment]
Nothing new yet, waiting for the review 😊

**Review by dfernandes-wanadev — APPROVED** [review]

**Comment by AlexOstrovsky** [comment]
Hi, any move here? Can I help with something?

**Comment by alexandre-daubois** [comment]
Unfortunately, Symfony 7 has been out now for 7-8 months so we decided to switch to 
[petitpress/gps-messenger-bundle](https://packagist.org/packages/petitpress/gps-messenger-bundle) (as we only use the Google Pubsub part of enqueue). I still hope this gets merged at some point for all the other projects out there that rely on Enqueue, but for our case, we sadly had to find another package to keep up with Symfony release process 😕

**Comment by makasim** [comment]
https://github.com/php-enqueue/enqueue-dev/pull/1362

**Pull request closed without merging**